### ISSUE TITLE
Can't connect to database with a username shorter than 4 characters

### ISSUE THREAD

**Issue by ak-coram** [emarsden/pgmacs#7]
This errors with the following trace:
```
Debugger entered--Lisp error: (args-out-of-range "ak" 0 4)
  seq-subseq("ak" 0 4)
  cl-subseq("ak" 0 4)
  pg-do-startup(#<PostgreSQL connection to postgres, pid nil> "postgres" "ak" "")
  pg-connect("postgres" "ak" "" "localhost" 5432 nil)
  #f(compiled-function (&rest ignore) #<bytecode 0x870aed5d27095f1>)((push-button :args nil :value "Connect" :notify #f(compiled-function (&rest ignore) #<bytecode 0x870aed5d27095f1>) :button-overlay #<overlay from 291 to 298 in *PGmacs connection widget*> :from #<marker (moves after insertion) at 291 in *PGmacs connection widget*> :to #<marker at 298 in *PGmacs connection widget*>) (push-button :args nil :value "Connect" :notify #f(compiled-function (&rest ignore) #<bytecode 0x870aed5d27095f1>) :button-overlay #<overlay from 291 to 298 in *PGmacs connection widget*> :from #<marker (moves after insertion) at 291 in *PGmacs connection widget*> :to #<marker at 298 in *PGmacs connection widget*>) nil)
  widget-item-action((push-button :args nil :value "Connect" :notify #f(compiled-function (&rest ignore) #<bytecode 0x870aed5d27095f1>) :button-overlay #<overlay from 291 to 298 in *PGmacs connection widget*> :from #<marker (moves after insertion) at 291 in *PGmacs connection widget*> :to #<marker at 298 in *PGmacs connection widget*>) nil)
  widget-apply-action((push-button :args nil :value "Connect" :notify #f(compiled-function (&rest ignore) #<bytecode 0x870aed5d27095f1>) :button-overlay #<overlay from 291 to 298 in *PGmacs connection widget*> :from #<marker (moves after insertion) at 291 in *PGmacs connection widget*> :to #<marker at 298 in *PGmacs connection widget*>) nil)
  widget-button-press(291)
  funcall-interactively(widget-button-press 291)
  command-execute(widget-button-press)
```

**Comment by ak-coram** [comment]
I just realized I should probably have created this for `pg-el` instead: please let me know if I should do that instead.

**Comment by emarsden** [comment]
Well, that was a pretty stupid bug! 

Thanks for the report, fixed in pg-el v0.43.
